content/en/docs/ops/deployment/platform-requirements/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Platform Requirements
+description: Platform requirements for Istio.
+weight: 1
+skip_seealso: true
+keywords: [platform-setup,prerequisites]
+owner: istio/wg-environments-maintainers
+test: no
+---
```

Comment:
Should we have an alias here for the old link so that anyone who might have bookmarked the old link will at least get a page instead of a 404?

Reply:
Didn't know we could do that! Will take a look.

Reply:
https://istio.io/latest/docs/releases/contribute/front-matter/#required-front-matter-fields